codec_specs.md

```diff
@@ -817,6 +817,16 @@ The `BlockAddId` of the `BlockMore` containing these data **MUST** be 1.
 
 Initialization: none
 
+### A_ATRAC/AT1
+
+Codec ID: A_ATRAC/AT1
+
+Codec Name: Sony ATRAC1 Codec
+
+Description: The original ATRAC codec by Sony, mainly used in MiniDisc platforms. The core technical details on ATRAC1 can be found at https://www.minidisc.wiki/technology/atrac/aes. An example encoder can be found at https://github.com/dcherednik/atracdenc.
```

Comment:
You should use a more fornal way to link to external resources, in the form of [@?AtracAes] and a description in `rfc_backmatter_codec.md`. The same can be done for the example. Although IMO it can be dropped.

Looking at the linked documents. They don't provide an actual description of the bitstreams that can be expected to be found in each `A_ATRAC/AT1` frame. Is there a more bitstream based documentation that we could link to ? For example the [code you mention on CELLAR](https://github.com/cybercase/netmd-js/blob/ada9272a6f670ff7813b4405fd8444809c5ceb67/src/utils.ts#L339) is closer to what could be used to decypher the bitstream but it's not a documentation.

Reply:
> You should use a more fornal way to link to external resources,

Hmmm... We have a lot of this method in the doc, e.g. [for V_MS/VFW/FOURCC](https://github.com/ietf-wg-cellar/matroska-specification/blob/master/codec_specs.md#v_msvfwfourcc).

> For example the [code you mention on CELLAR](https://github.com/cybercase/netmd-js/blob/ada9272a6f670ff7813b4405fd8444809c5ceb67/src/utils.ts#L339) is closer to what could be used to decypher the bitstream but it's not a documentation.

We also have a lot of similar references lie that e.g. [for V_REAL/RV40](https://github.com/ietf-wg-cellar/matroska-specification/blob/master/codec_specs.md#v_realrv40).

IMO we should not request a lot more for just reserving an identifier, we understand well that this is ATRAC1 and player willing to support this format would have what they need for decoding it.

Reply:
> Hmmm... We have a lot of this method in the doc, e.g. [for V_MS/VFW/FOURCC](https://github.com/ietf-wg-cellar/matroska-specification/blob/master/codec_specs.md#v_msvfwfourcc).

The document is far from finished and this is the kind of thing we have to fix. So it's better if we don't introduce more of these.

> IMO we should not request a lot more for just reserving an identifier

IMO it's not good enough to have just an identifier and then good luck to find the relevant information to parse the data. This is especially true for rare codecs. The goal here is clearly sustainability and with a rare codec like that you can't expect people to easily be able to know how to use it.

Reply:
That being said, I'm fine with having a first step to reserve the codec ID so people are aware of it.

Reply:
Hi, I've been contributing to the reverse engineering project mentioned on the Cellar mailing list and can hopefully shed a little bit of light on where our understanding of ATRAC is. 

So far, our best sources of information have been the ATRAC decoder [included](https://github.com/FFmpeg/FFmpeg/blob/master/libavcodec/atrac1.c) in FFmpeg and the now expired Sony US patent [5301205](http://www.minidisc.org/patents/pdfs/US05301205.pdf). To the best of my knowledge, Sony never released a formal specification for the bitstream.

The ATRAC decoder in FFmpeg is - as far as we're aware - correct, and we'd like to make it easier to leverage that without the aforementioned issues with the AEA container.

Reply:
Hi,

As thinkbrown has mentioned, there does not exist any technical documentation on the bitstream format that is publicly available (as far as we know). The closest is the decoder source code in ffmpeg and atracdenc. The goal of getting the codec ID is to make the format more accessible. Potentially we could write this documentation, given time, but right now our focus is on the reverse engineering process and audio archival.

I will take a look at the referencing system and try to submit an update as soon as possible.

Reply:
Updated the references to conform to the suggested formatting.